pkg/ottl/ottlfuncs/README.md

```diff
@@ -542,6 +543,23 @@ Examples:
 
 - `IsString(attributes["maybe a string"])`
 
+### IsInt
+
+`IsInt(value)`
+
+The `IsInt` Converter returns true if the given value is a integer.
+
+The `value` is either a path expression to a telemetry field to retrieve or a literal. 
+
+If `value` is an `int64` or `pcommon.ValueTypeInt` then returns `true`, otherwise returns `false`.
```

Comment:
Looks like int8, etc also result in a true return

Reply:
Although OTTL basically forces int64, its true that the function is less strict, so we can make the README reflect reality like @strawgate mentions.